apps/arm/src/main.cpp

```diff
@@ -167,14 +168,75 @@ static mbed_error_status_t setToolTipDeployment(CANMsg &msg) {
   }
 }
 
-// Enable or disable PID tuning mode10
+// Enable or disable PID tuning mode
+// Manually send CAN message to SET_PID_TUNING_MODE with appropriate boolean
 static mbed_error_status_t setPIDTuningMode(CANMsg &msg) {
-  return MBED_SUCCESS;  // TODO
+  bool allowPIDTuning;
+  bool updateFailed = false;
+  msg.getPayload(allowPIDTuning);
+  if (allowPIDTuning) {
+    updateFailed |= turnTableActuator.allowPIDTuning();
+    updateFailed |= shoulderActuator.allowPIDTuning();
+    updateFailed |= elbowActuator.allowPIDTuning();
+    updateFailed |= wristLeftActuator.allowPIDTuning();
+    updateFailed |= wristRightActuator.allowPIDTuning();
+    updateFailed |= clawController.allowPIDTuning();
+  } else {
+    updateFailed |= turnTableActuator.disallowPIDTuning();
+    updateFailed |= shoulderActuator.disallowPIDTuning();
+    updateFailed |= elbowActuator.disallowPIDTuning();
+    updateFailed |= wristLeftActuator.disallowPIDTuning();
+    updateFailed |= wristRightActuator.disallowPIDTuning();
+    updateFailed |= clawController.disallowPIDTuning();
+  }
+  return updateFailed ? MBED_ERROR_MUTEX_LOCK_FAILED : MBED_SUCCESS;
 }
 
 // Configure PID parameters
 static mbed_error_status_t setPIDParameter(CANMsg &msg) {
-  return MBED_SUCCESS;  // TODO
+  printf("Received request to update PID params over CAN\n");
+  /** ------------------------------------
+   * | 32 bits  | 8 bits   |  8 bits     |
+   * | data     | vel/pos  |  Actuator ID|
+   *  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^*/
+  HWBRIDGE::ARM::PID::tuningApiPayload payload;
+  msg.getPayload(payload);
+  printf("payload: %s", HWBRIDGE::ARM::PID::str(payload).c_str());
+  LookupTable<HWBRIDGE::ARM::ACTUATORID, std::reference_wrapper<ActuatorController>> actuatorIDLUT = {
```

Comment:
i felt like the LUT was just gonna uselessly take up memory since it is not used at mission time. tbf our micro prob wont even notice lol. whats the downside to it being inside a function? is allocating and deallocating every time too slow